custom-words.txt

```diff
@@ -5,6 +5,7 @@
 Bitwarden
 bytemark
 CODEOWNERS
+codeownership
```

Comment:
⚠️ We have this as two words everywhere I've seen.

Reply:
👍 Reverted in [1603713](https://github.com/bitwarden/contributing-docs/pull/259/commits/1603713e09f0bca5c6f0f1d70eab11b577f7d1f9)